<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
*{
color: #78799c;
border: none;
}

#frame_2, #frame_5{
	background-color: #2a2c49;
}

#mainBody,#Form, #frame{
	background-color: #1a1f39;
}

QLineEdit{
	background-color:transparent;
}

#searchFrame, QLineEdit{
	border-radius: 10px;
	border: 2px solid #78799c;
}

#appHeader{
	color:#78799c;
}

#prodottiHeader{
	color:  #a06ce3;
}

#avantiarw{
	border: 1px solid #a06ce3;
	background-color: #a06ce3 ;
	border-radius: 5px;
	
}

#finestreSecondarie{
	background-color: #2a2c49;
	border: solid;
	border-radius: 20px;
}

QLabel{
	color:#78799c;
}

#mainFrame{
	background-color: #2a2c49;
}

#homeBtn{
	color:#78799c;
	background-color:#1a1f39;
	padding : 10px 5px;
	text-align:left;
	border-top-left-radius:25px;
}

QPushButton{
	padding : 10px 5px;
	text-align:left;
	color:#78799c;
	font: 10pt &quot;Leelawadee UI&quot;;
}


#frame_3{
	background-color:#2a2c49;
	border: solid;
	border-radius: 20px;
}
QTableWidget{
	background-color:#2a2c49;
	border: solid;
}
#leftMenu,#frame{
background-color: #2a2c49;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="applicationName">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Usato&lt;/span&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#aaaaff;&quot;&gt;Beato&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLineEdit" name="emailEd">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>EMAIl</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLineEdit" name="passwordEd">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>PASSWORD</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="confermaBtn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Conferma</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
